sellercenter-setting/src/main/java/com/gfg/sellercenter/setting/service/SettingService.java

```diff
@@ -0,0 +1,35 @@
+package com.gfg.sellercenter.setting.service;
+
+import com.gfg.sellercenter.setting.entity.Setting;
+import com.gfg.sellercenter.setting.reader.HttpReader;
+import lombok.AllArgsConstructor;
+import org.json.JSONObject;
+
+@AllArgsConstructor
+public class SettingService implements SettingServiceInterface {
+
+    private HttpReader httpReader;
+
+    public Setting getSetting(String field, int sellerId) throws Exception
```

Comment:
I would instead of overloading here use different method name to be more explicit, like getSellerSetting()

Reply:
It actually doesn't overload, since it doesnt extend `HttpReader`. 

In general I don't mind calling it just getSetting, (although I think the more common name would be something like `getValue` or `getSettingValue`, but I would really like to have some normal explanation what does the class and the method do with javadoc. That way you would hover the method and it will tell you, ```
gets the seller center setting value exposed by (currently) graph service```

Reply:
> It actually doesn't overload, since it doesnt extend `HttpReader`.
> 
> In general I don't mind calling it just getSetting, (although I think the more common name would be something like `getValue` or `getSettingValue`, but I would really like to have some normal explanation what does the class and the method do with javadoc. That way you would hover the method and it will tell you, ` gets the seller center setting value exposed by (currently) graph service`

This method will not return a value, it just returns the setting with the important information back
SellerId, Field and the actual Value of the field so i think getSetting is the best fitting name of the function.
With overload @mmirboboev meant that i overvload my getSetting method inside of SettingService

Reply:
Ah, ok. Then `getSetting` is a good name, and personally I don't see a reason to not overload. The javadoc would solve the problem of explicity *wink *wink*